include/seastar/parquet/parquet_types.h

```diff
@@ -0,0 +1,2915 @@
+/**
+ * Autogenerated by Thrift Compiler (0.13.0)
```

Comment:
Ok - so this one should either be generated during building or we need to set a version check in CMake, as mentioned above in my comment for FindThrift.

Reply:
Fixed in 6cbc79e.